evento/src/views/Explore/index.js

```diff
@@ -20,7 +20,9 @@ class Explore extends Component {
 	async componentDidMount() {
 		const result = await api.getEvents();
```

Comment:
Since you are changing this code, move the contents of componentDidMount to it's own function already. fetchUpcomingEvents() or fetchEvents()